src/internal/provider/update.go

```diff
@@ -77,11 +82,15 @@ func (p Provider) getSemVerTagsFromRSS() ([]string, error) {
 func (p Provider) getLastSemVerTag() (string, error) {
 	semverTags, err := p.getSemVerTagsFromRSS()
 	if err != nil {
-		return "", err
+		p.Logger.Error("Unable to fetch tags, skipping", slog.Any("err", err))
+		return "", nil
```

Comment:
I would consider returning a "known error" here that can be checked with `errors.Is` instead of returning an empty tag with no error. From a Go convention standpoint I expect the function to return something meaningful if no error happened, which doesn't happen here. This seems like it should be handled on the caller side.

Reply:
I was keeping with the current "pattern" in the codebase that doesn't use custom error types.  I'm hoping to rewrite a lot of this code and was trying to keep the changeset minimal.

Reply:
Not saying it's a good pattern...